### PR TITLE
feat:products page added

### PR DESCRIPTION
**Pull request by ArpitPatni**
Please review my products page!
Thank you!

### CODE REVIEW

**Comment by ArpitPatni** [comment]
Demo link: 
https://shoeestore.netlify.app

File: src/Pages/Products/Products.css [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    font-weight: bolder;
+    font: 1.2em sans-serif;
+  }
+  
+  .range {
+    display: flex;
+    justify-content: space-around;
+    font: 1em sans-serif;
+    color: rgb(92, 90, 90);
+  }
```
Comment (bhavyaj12) [inline src/Pages/Products/Products.css:51]: Try to use global css variables for colors. And also stick to one type of colour convention, so for example only hex codes or only rgba

File: src/Components/FilterSideBar/Filters.jsx [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+                        className="ecom-btn"
+                        onClick={() => {
+                            FilterDispatch({ type: "Clear_Filter" });
+                          }}
+                    >
+                        Clear
+                    </button>
+                </div>
+                <div className="filter-cat">
+                    <span>
```
Comment (bhavyaj12) [inline src/Components/FilterSideBar/Filters.jsx:26]: In filters, if only one can be selected then use radio button not checkbox. For filter by category, multi select should be there, right now able to select only one. Also star rating should be shown on the card.

**Review by bhavyaj12 — APPROVED** [review]
UI is clean, code logic is easy to understand.
Suggested some changes. 
Rest, good work

**Review by krgarima — COMMENTED** [review]
Some suggestions: 
1. Fix this
ar1
2. You can lighten the color here (optional). It will look better.
ar2
3. Increase the top margin of `Add to Cart` and `Add to wishlist` button.

File: src/Pages/Products/Products.css (hunk 1 above)
Comment (ArpitPatni) [reply src/Pages/Products/Products.css:51]: thank you for the suggestion!

**Pull request merged**